src/Sarif.Sarifer/AnalyzeProjectCommand.cs

```diff
@@ -15,14 +16,17 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Sarifer
 {
-    internal class AnalyzeProjectCommand
+    internal class AnalyzeProjectCommand : IDisposable
     {
         private DTE2 dte;
         private IComponentModel componentModel;
         private IBackgroundAnalysisService backgroundAnalysisService;
+        private readonly CancellationTokenSource cancellationTokenSource;
+        private bool disposedValue;
```

Comment:
>disposedValue [](start = 21, length = 13)

suggest just using 'disposed' for this variable or 'isDisposed' if that seems clearer to you.

Reply:
good idea. just changed to `disposed` in all places that i implemented `IDisposable`

---
In reply to: [532103867](https://github.com/microsoft/sarif-visualstudio-extension/pull/291#discussion_r532103867) [](ancestors = 532103867)